contrib/checkbox-provider-ce-oem/units/test-plan-ce-oem.pxu

```diff
@@ -76,6 +77,7 @@ nested_part:
     ce-oem-ethernet-tcp-automated
     com.canonical.certification::eeprom-automated
     com.canonical.certification::rtc-automated
+    ce-oem-iio-sensors-automated
```

Comment:
I think it should be `before-suspend`?

Reply:
you're right, thanks for point out this issue.

Reply:
```suggestion
    before-suspend-ce-oem-iio-sensors-automated
```